membership-attribute-service/app/services/IdentityService.scala

```diff
@@ -1,43 +0,0 @@
-package services
-
-import com.typesafe.scalalogging.LazyLogging
-import play.api.libs.json.{JsValue, Json, __}
-import java.net.URLEncoder.encode
-
-import IdentityService._
-import com.gu.okhttp.RequestRunners
-import okhttp3.{MediaType, Request, RequestBody, ResponseBody}
-
-import scala.concurrent.{ExecutionContext, Future}
-
-object IdentityService {
```

Comment:
Might we need this later in the week to do the saves impression tracking?

Reply:
I had assumed that the saves tracking would be done using GA? Perhaps we can discuss offline if I've misunderstood?

Reply:
I was thinking more about automating what @lmath has been doing via ELK analysis. 

Also FYI, we must no longer put Identity ID into GA.

Reply:
OK, I guess we can discuss further once we get to that card. For now I think I'll just remove this code and we can re-instate it if we need it later...